conftest.py

```diff
@@ -50,14 +50,10 @@ def notification(player):
 
 
 @pytest.fixture
-def game_event(court, court_ball_game):
-    return GameEvent.objects.create(
-        id=20,
-        time=timezone.now(),
-        level_of_game=5,
-        min_number_of_players=3,
-        max_number_of_players=5,
-        court=court, ball_game="Basketball")
+def game_event(court):
+    return GameEvent.objects.create(id=2, time=timezone.now(), level_of_game=3,
+                                    min_number_of_players=2, max_number_of_players=5,
+                                    court=court, ball_game=BallGame.Baseball)
```

Comment:
I think it was more readable before. I think the cleanest is this way:
```suggestion
def game_event(court):
    return GameEvent.objects.create(
        id=2,
        time=timezone.now(),
         level_of_game=3,
         min_number_of_players=2,
         max_number_of_players=5,
         court=court,
         ball_game=BallGame.Baseball,
    )
```
This way, each argument is on a line of its own, and each line ends with a comma, so if you add another line at later time, you don't have to change any existing line - easier to review.